basilisk/gui/conversation_tab.py

```
@@ -164,7 +168,9 @@ def __init__(
 		self.last_time = 0
 		self.recording_thread: Optional[RecordingThread] = None
 		self.task = None
+		self.process: Optional[Process] = None
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)